copter/source/docs/traditional-helicopter-autotune.rst

```diff
@@ -49,6 +49,8 @@ User can specify the tuning desired using the :ref:`AUTOTUNE_SEQ<AUTOTUNE_SEQ>`
 |                                                     | +---------+----------------------------------------+|
 |                                                     | | 8       | Max Gain Determination                 ||
 |                                                     | +---------+----------------------------------------+|
+|                                                     | | 16      | Tune Check                             ||
+|                                                     | +---------+----------------------------------------+|
 +-----------------------------------------------------+-----------------------------------------------------+
 
 ** Max gain determination is always done before tuning Rate D and Rate P gains. This test determines the maximum allow values of ``ATC_RAT_xxx_D`` & ``ATC_RAT_xxx_P`` so that the helicopter never experiences servere oscillations due to raising these gains too high.  Since the maximum allowable values of ``ATC_RAT_xxx_D`` & ``ATC_RAT_xxx_P`` are found, tuning is done by incrementing these gains by 5% of their maximum allowable value.
```

Comment:
Evidently combinations of bits 3 or 4 with the others are not allowed...are there code protections?in any case, it should be mentioned that bit 3 or 4 must be set without any other bits, or the precedence explained...

Reply:
@Hwurzburg bits 3 and 4 just complementary tests. Bit 3 is done as part of the rate D and rate P test.  Bit 3 is just provided for people that just want to see what the max gains are without actually tuning. Bit 4 is provided for people that want to evaluate the final tune. It can be combined with others without harm.

Reply:
then we should state that...the values listed imply otherwise

Reply:
just to be clear, any or all of the bits can be set in any combination

Reply:
Yes that is true but it doesn't make sense from a tuning perspective to do it that way.  The combinations that I have shown to this point are combinations that I would expect to be used.  The third bit is just for information.  So it could be used with lets say the FF test if the user just wanted to investigate the limit of the rate P and D gains after tuning the FF gain.  You would not use it with the Rate P and D gain because it is already part of this test.

I would really prefer you just clarify what you meant above for the 3rd and 4th bits and leave the other combinations as they are.

Reply:
I think we need to be clearer about when or when not you should set bit 3 in the MAX GAIN section....and what to look at in the logs for TUNE CHECK....is it just the items in the LOG Analysis section? if so, we need to directly state that in the TUNE CHECK section